operate/client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "camunda-operate",
-  "version": "8.6.10-SNAPSHOT",
+  "version": "8.6.13-SNAPSHOT",
```

Comment:
❓ Interesting. Why does this appear as a change here? 😄 Was this overlooked in any previous release or multiple of them? Should we do something about that? 🙂

Reply:
Thats a good question, to be honest, I dont know the answer, once it comes from Front-end 

I am going to gather some help here in order to understand why and what is the impact of keeping this here.

Reply:
Cool, thank you! 👍

Reply:
Thank you @vsgoulart :)